package.json

```diff
@@ -22,5 +22,9 @@
     "pkgroll": "^2.12.2",
     "turbo": "^2.5.3",
     "typescript": "^5.8.3"
+  },
+  "dependencies": {
+    "file-saver": "^2.0.5",
+    "jszip": "^3.10.1"
```

Comment:
These can be moved to `devDependencies`. Because we're using `vite` to bundle everything at dev time we don't actually have runtime dependencies as far as the tooling is concerned.